frontend/src/scenes/project/CreateProjectModal.tsx

```diff
@@ -31,6 +32,7 @@ export function CreateProjectModal({
     const handleSubmit = (): void => {
         createTeam({ name, is_demo: false })
         reportProjectCreationSubmitted(currentOrganization?.teams ? currentOrganization.teams.length : 0, name.length)
+        loadCurrentOrganization()
```

Comment:
Currently this is a race condition, because we are calling the non-blocking version of `createTeam`. We could use `useAsyncActions(organizationLogic)` for it instead, and then call `await createTeam(...)` in this function, so that by the time we call `loadCurrentOrganization()`, we're sure that the new project actually exists by now.

However, this code basically connects two logics – it really belongs _in_ the logics. This way if we ever call `createTeam` anywhere else, we won't have to _remember_ about `loadCurrentOrganization()`.
Let's call `loadCurrentOrganization` via a listener in `teamLogic`. Specifically, it should be a `createTeamSuccess` listener (every Kea loader action has `*Success` and `*Failure` companions, which are fired when loading finishes).